modules/monitoring-specifying-how-a-service-is-monitored.adoc

```diff
@@ -47,6 +47,11 @@ spec:
 +
 This defines a `ServiceMonitor` resource that scrapes the metrics exposed by the `prometheus-example-app` sample service, which includes the `version` metric.
 
+[NOTE]
+====
+For a service in a user-defined project, the service monitor can discover the `/metrics` endpoint only in the namespace specified in the `ServiceMonitor` resource. Therefore, you cannot create a `ServiceMonitor` resource for a namespace and use the `namespaceSelector` field to monitor additional namespaces.
```

Comment:
`you cannot create a `ServiceMonitor` resource for a namespace and use the `namespaceSelector` field to monitor additional namespaces`, `you cannot` is improper, should be `you need not to`.
since ignoreNamespaceSelectors is true, then the servicemonitor and podmonitor settings are overridden and prometheus is configured to only look up resources in the current namespace.
```
# oc -n openshift-user-workload-monitoring get prometheus/user-workload -oyaml | grep ignoreNamespaceSelectors
  ignoreNamespaceSelectors: true
```
see the test result from
http://pastebin.test.redhat.com/971992

Reply:
@juzhao is correct, maybe it should be phrased like: "A ServiceMonitor resource in a user-defined namespace can only discover services in the same namespace. In other words, the`namespaceSelector` field of the ServiceMonitor resources will always be ignored."

Reply:
@simonpasquier @juzhao Thank you for the review! I applied your feedback if you can PTAL and approve if everything looks good now.